package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dtrace-provider",
-  "version": "0.8.5",
+  "version": "0.8.6",
```

Comment:
Should it be at least a minor version bump, since we add support for new environment variables?

Reply:
The goal of this change is to be a minor bug fix to improve the installation experience for consumers of packages that use node-dtrace-provider, like bunyan and restify, which use "~0.8".

Reply:
But at the same time, a module relying on a ~0.8 dependency _and_ using one of the new env var wouldn't be guaranteed to have that new env var supported for every install, is that correct?

Reply:
That's right. If they're using the env var set to `hard`, they'll either have the message-printing behaviour at runtime, or the module will fail to build. I think that a large part of the problem here is that `dtrace-provider` needs to be released past `1.0` so that libraries can depend on `^1.0.0` and additions like this can be a minor bump. I can publish `0.8.6` and then publish `1.0.0` so that consumers can begin updating their `package.json` dependencies.